$OpenBSD: patch-qucs-core_src_math_complex_h,v 1.1 2017/05/02 15:13:39 espie Exp $
Don't include namespace std with libcxx, ambiguity with vector.
Don't multi-declare polar functions, especially ones we don't define
(stolen from qucs-current)

Index: qucs-core/src/math/complex.h
--- qucs-core/src/math/complex.h.orig
+++ qucs-core/src/math/complex.h
@@ -37,7 +37,6 @@ using namespace std::tr1;
 typedef std::complex<nr_double_t> nr_complex_t;
 #else
 #include <complex>
-using namespace std;
 typedef std::complex<nr_double_t> nr_complex_t;
 #endif
 
@@ -118,8 +117,6 @@ nr_complex_t   polar (const nr_double_t mag, const nr_
 
 #ifndef HAVE_CXX_COMPLEX_POLAR_COMPLEX
 nr_complex_t   polar (const nr_complex_t a, const nr_complex_t p);
-nr_complex_t   polar (const nr_double_t a, const nr_complex_t p);
-nr_complex_t   polar (const nr_complex_t a, const nr_double_t p = 0.0);
 #endif
 
 #ifndef HAVE_CXX_COMPLEX_POW
